src/Compilers/VisualBasic/Portable/Symbols/EventSymbol.vb

```diff
@@ -207,7 +207,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                         ' TODO: Perhaps the error wording could be changed a bit to say "type of event..." ?
                         '
                         ' Reference required to assembly '{0}' containing the definition for event '{1}'. Add one to your project.
-                        errorInfo = ErrorFactory.ErrorInfo(ERRID.ERR_UnreferencedAssemblyEvent3, errorInfo.Arguments(0), Me)
+                        errorInfo = ErrorFactory.ErrorInfo(ERRID.ERR_UnreferencedAssemblyEvent3,
+                                                           CustomSymbolDisplayFormatter.DefaultErrorFormatIfErrorSymbol(errorInfo.Arguments(0)),
+                                                           CustomSymbolDisplayFormatter.DefaultErrorFormatIfErrorSymbol(Me))
```

Comment:
Why is this change needed, it doesn't look like ERR_UnreferencedAssemblyEvent3 was specially handled in ConvertSymbolToString?